.github/workflows/continuous-integration.yml

```diff
@@ -40,14 +40,14 @@ jobs:
 
       - name: "Run PHPUnit"
         continue-on-error: true
-        run: "vendor/bin/phpunit -c ci/github/phpunit/sqlite.xml --coverage-clover=coverage-no-cache.xml"
+        run: "vendor/bin/phpunit -c ci/github/phpunit/sqlite.xml --coverage-clover=coverage-no-cache.xml --exclude-group performance,locking_functional,dbal2"
```

Comment:
@greg0ire Are `performance` and `locking_functional` not needed for the regular "Run PHPUnit" step?

Reply:
`performance` is run separately, that's for sure. As for `locking_functional`, it's about Gearman, so it might be great to run it someday. They are already excluded in `sqlite.xml`, but I think @scyzroyck had to respecify them in order to add `dbal2` to the list.

Reply:
We'll see how this will affects the code coverage in case some test won't be executed anymore.

Reply:
[looks like it does](https://github.com/doctrine/orm/pull/8898#discussion_r693210346)